src/main/java/org/broadinstitute/hellbender/engine/VariantAdapter.java

```diff
@@ -0,0 +1,35 @@
+package org.broadinstitute.hellbender.engine;
+
+import htsjdk.variant.variantcontext.VariantContext;
+
+/**
+ * VariantAdapter wraps the existing htsjdk VariantContext class so it can be
+ * used with the Variant API.
+ */
+public class VariantAdapter implements Variant {
+    private VariantContext variantContext;
+    @Override
+    public int getStart() {
+        return variantContext.getStart() - 1;
+    }
+    @Override
+    public int getEnd() {
+        return variantContext.getEnd() - 1;
```

Comment:
We want 1-based inclusive end positions, which is what `variantContext.getEnd()` returns, so don't subtract 1 here.